country-a-service/cda-generator/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/cda/model/EPrescriptionL3.java

```diff
@@ -24,14 +24,14 @@ public class EPrescriptionL3 {
 
     /**
      * The.. (this).. <effectiveTime> element encodes the start and stop time of the medication regimen or the length of the medication regimen. This an interval of time (xsi:type='IVL_TS'), and must be specified as shown. This is an additional constraint placed upon CDA Release 2.0 by this profile, and simplifies the exchange of start/stop/length and frequency information between EMR systems. If no information is available for the dosage period, a nullFlavor attribute has to be provided with the value 'UNK'.
-     *
+     * <p>
```

Comment:
Didn't you set JD_P_AT_EMPTY_LINES to false?  I would've guessed that this is the behaviour when `true`.  Not that I have a strong opinion either way atm.